system_test.sh

```diff
@@ -40,7 +40,7 @@ trap "$clear_tty; $clear_tmux; $clear_tmp" EXIT
 
 utils="file pgrep git vim tmux"
 for util in $utils; do
-	p=$(which $util 2> /dev/null) || {
+	p=$(command -v $util 2> /dev/null) || {
```

Comment:
no need to redirect `stderr`

Reply:
true, I'll remove the redirection.

Regarding to `command -v` and aliases, we can change it to `command -pv`, where `-p` should be also commonly supported but
> If the `-p` option is given, the search for command is performed using a default value for `PATH` that is guaranteed to find all of the standard utilities.

The difference in my case is:
```
$ command -v ls
alias ls='ls --color=auto'
```
and
```
$ command -pv ls
/bin/ls
```

Reply:
I just have force pushed update with removed `stderr` redirection and appended `-p` in order to avoid `alias` outputs.